docs/lineage/dagster.md

```diff
@@ -0,0 +1,82 @@
+# Dagster Integration
+DataHub supports the integration of
+
+- Dagster Pipeline metadata
+- Job and Op run information as well as
+- Lineage information when present
+
+## Using Datahub's Dagster Sensor
+
+Dagster sensors allow us to perform some actions based on some state change. Datahub's defined dagster sensor will emit metadata after every dagster pipeline run execution. This sensor is able to emit both pipeline success as well as failures. For more details about Dagster sensors please refer [Sensors](https://docs.dagster.io/concepts/partitions-schedules-sensors/sensors).
+
+### Prerequisites
+
+1. You need to create a new dagster project. See <https://docs.dagster.io/getting-started/create-new-project>.
+2. There are two ways to define Dagster definition before starting dagster UI. One using [Definitions](https://docs.dagster.io/_apidocs/definitions#dagster.Definitions) class (recommended) and second using [Repositories](https://docs.dagster.io/concepts/repositories-workspaces/repositories#repositories).
+3. Creation of new dagster project by default uses Definition class to define Dagster definition.
+
+### Setup
+
+1. You need to install the required dependency.
+
+```shell
+pip install acryl_datahub_dagster_plugin
+```
+
+2. You need to import DataHub dagster plugin provided sensor definition and add it in Dagster definition or dagster repository before starting dagster UI as show below:
+**Using Definitions class:**
+
+```python
+from dagster import Definitions
```

Comment:
our docs site has support for inlining files - might be nicer to inline an example file instead of embedding it here